node/node.go

```diff
@@ -98,16 +98,17 @@ type nodeChainSyncer interface {
 	HandleNewTipSet(ctx context.Context, ci *types.ChainInfo, trusted bool) error
 }
 
+
 // Node represents a full Filecoin node.
 type Node struct {
 	host     host.Host
 	PeerHost host.Host
 
-	Consensus    consensus.Protocol
-	ChainReader  nodeChainReader
-	MessageStore *chain.MessageStore
-	Syncer       nodeChainSyncer
-	PowerTable   consensus.PowerTableView
+	Consensus           consensus.Protocol
```

Comment:
nit: this seems incorrect wrt standard linting.

Reply:
Yes, to put it succinctly I had some environment issues and had to redo my git clone and pre-commit hooks, so it should be fixed now.